example/package.json

```diff
@@ -12,12 +12,10 @@
     "@emotion/react": "^11.1.5",
     "@emotion/styled": "^11.3.0",
     "framer-motion": "^5.3.3",
-    "react-app-polyfill": "^2.0.0"
-  },
-  "alias": {
-    "react": "../node_modules/react",
-    "react-dom": "../node_modules/react-dom/profiling",
-    "scheduler/tracing": "../node_modules/scheduler/tracing-profiling"
```

Comment:
Is the alias thing not working? I guess I switched from parcel to esbuild or something...

Reply:
I didn't find anything about this alias thing. where does it come from? on my local with fresh pull it didn't work.